graphql.md

```diff
@@ -278,6 +279,7 @@ type CertifyGood {
   justification: String!
   origin: String!
   collector: String!
+  knownSince: Time!
```

Comment:
@nathannaveen Missing this on `hasSBOM`

Reply:
Fixed here: https://github.com/guacsec/guac-docs/pull/102.